src/Components/API/AppiCalls.js

```diff
@@ -1,18 +1,23 @@
 export const ApiData = {
 fetchAll: () => {
```

Comment:
I find this method of API calls really cool. I like that everything is in one big object. 🤔